Int[]	availableDirections
Int	previousAutoID
Int	nextCrossID
Int	distanceToNextCross
